semver/__init__.py

```diff
@@ -785,15 +785,15 @@ def parse_range(self, range_):
         set_ = [make_comparator(comp, loose) for comp in set_]
         return set_
 
-    def test(self, version):
+    def test(self, version, include_prerelease=False):
         if not version:  # xxx
             return False
 
         if isinstance(version, str):
             version = make_semver(version, loose=self.loose)
 
         for e in self.set:
-            if test_set(e, version):
+            if test_set(e, version, include_prerelease):
```

Comment:
This is tiny thing, sorry..
I like `include_prerelease=include_prerelease`, even arguments is not keyword only arguments.

Reply:
No problem, changed it!